app/views/groups/create/clients/review_clients_paginated.scala.html

```diff
@@ -49,7 +49,13 @@
             Seq(
                 Html(client.name),
                 Html(displayObfuscatedReference(client.name, client.hmrcRef)),
-                Html(displayTaxServiceFromServiceKey(client.taxService))
+                Html(displayTaxServiceFromServiceKey(client.taxService)),
+                Html(
+                    a(
+                        key = msgs("common.remove"),
+                        href = routes.CreateGroupSelectClientsController.showConfirmRemoveClient(client.id).url
+                    ).toString
```

Comment:
You don't need to have `a().toString` wrapped in `Html()` - could just have `a()` :D